sql/solutions/lesson3.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: page
+title: Introduction to SQL Solutions
+---
+# Lesson 3 solutions to exercises
+
+## L3.1 How many people are not living in their country of birth.
+
+```SQL
+SELECT COUNT(*)
+  FROM people
+ WHERE country_of_birth IS NOT country_of_residence;
```

Comment:
I just learnt something. I didn't know this was possible X IS NOT Y... I always did it like this:

SELECT COUNT(*)
  FROM people
 WHERE country_of_birth != country_of_residence;

or 

SELECT COUNT(*)
  FROM people
 WHERE country_of_birth <> country_of_residence;

Can you add these two alternatives too?